AntistasiOfficial.Altis/Stringtable.xml

```diff
@@ -1581,724 +1957,894 @@
 		<Container name="REINIT_Y">
 		     <Key ID="STR_HINTS_RY">
 			    <Original>Reinitialised:\n\nSpecial Keys\n\nStatistics Report</Original>
 				<Russian>Переинициализировано:\n\Специальные клавиши\n\n Отчет о статистике</Russian>
+			     	<Spanish>Reinicializado:\n\nTeclas Especiales\n\nInforme Estadísticas</Spanish>
 			</Key>
 		</Container>
 		<Container name="SERVER_AUTOSAVE">
 		     <Key ID="STR_HINTS_SERV_AUTO_SAVE">
 			    <Original>A critical failure has been detected in Antistasi. Please contact Admin and ask for a diagnose + server restart. Apologies.</Original>
 				<Russian>В Antistasi обнаружен критический сбой. Пожалуйста, свяжитесь с администратором и попросите диагностировать + перезапустить сервер. Простите за неудобство.</Russian>
+			     	<Spanish>Se ha detectado un fallo crítico en Antistasi. Por favor, contacte con el Admin y pregunte por un diagnóstico + reiniciar. Disculpas.</Spanish>
 			</Key>
 		</Container>
 		<Container name="SKIP_TIME">
 		     <Key ID="STR_HINTS_ST_OTCCOTR">
 			    <Original>Only the Commander can order to rest</Original>
 				<Russian>Только Командир может отдать приказ отдыхать</Russian>
+			     	<Spanish>Sólo el Comandante puede ordenar descanso</Spanish>
 			</Key>
 			<Key ID="STR_HINTS_ST_YCRWENOU">
 			    <Original>You cannot rest with enemies near our units</Original>
 				<Russian>Вы не можете отдыхать когда противник находится рядом с Вашим отрядом</Russian>
+				<Spanish>No puedes descansar con enemigos cerca de nuestras unidades</Spanish>
 			</Key>
 			<Key ID="STR_HINTS_ST_YCRWAAFOCSATIC">
 			    <Original>You cannot rest while AAF or CSAT is counterattacking</Original>
 				<Russian>Вы не можете отдыхать, когда AAF или CSAT контратакуют</Russian>
+				<Spanish>No puedes descansar mientras las AAF o el CSAT están contraatacando</Spanish>
 			</Key>
 			<Key ID="STR_HINTS_ST_YCRWYHQIUA">
 			    <Original>You cannot rest while your HQ is under attack</Original>
 				<Russian>Вы не можете отдыхать, пока Штаб атакуют</Russian>
+				<Spanish>No puedes descansar mientras el HQ está bajo ataque</Spanish>
 			</Key>
 			<Key ID="STR_HINTS_ST_APMBIA100MRFHQ">
 			    <Original>All player must be in a 100mts radius from HQ to be able to rest</Original>
 				<Russian>Все игроки должны находиться в радиусе 100 м от Штаба, чтобы иметь возможность отдыхать</Russian>
+				<Spanish>Todos los jugadores deben estar en un radio de 100m del HQ para Descansar</Spanish>
 			</Key>
 		</Container>
 		<Container name="TELEPORT">
 		     <Key ID="STR_HINTS_TELEPORT">
 			    <Original>Click on the position you wish to teleport to.</Original>
 				<Russian>Кликните по зоне, куда Вы хотите телепортироваться.</Russian>
+			     	<Spanish>Click en la posición a la que deseas ser teletransportado.</Spanish>
 			</Key>
 		</Container>
 		<Container name="GARBAGE_CLEAR">
-			<Key ID="STR_HINTS_GARB_CLR">
-				<Original>Deleting Garbage. Please wait</Original>
+		     <Key ID="STR_HINTS_GARBAGE_CLR">
```

Comment:
you can not do this

Reply:
Ok, I see what u say. Thx, change this part